BE/teamgu/src/main/java/com/teamgu/api/controller/AdminController.java

```diff
@@ -149,5 +150,18 @@
 		return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
 	}
 	
+	@ApiOperation(value = "DashBoard Table 조회")
+	@GetMapping("/dashboardtable/{projectId}")
+	public ResponseEntity<? extends BasicResponse> getTeamBuildingTable(@PathVariable Long projectId) {
+
+		if(!adminService.checkProjectDeletion(projectId)) {
+
+			List<DashBoardTableResDto> dashBoardTable = adminService.getDashBoardTableInfo(projectId);
+			
+			return ResponseEntity.ok(new CommonResponse<List<DashBoardTableResDto>>(dashBoardTable));
+		}
+		return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
```

Comment:
```suggestion
		return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
		return ResponseEntity.status(HttpStatus.BAD_REQUEST)
                    .body(new ErrorResponse("입력값이 정확하지 않습니다"));
```
어떤 것 때문에 에러가 발생하는지 응답 메세지에 포함되면 좋겠습니다

Reply:
Admin Controller 응답 메세지 포함 반영 완료되었습니다.